addons/web_editor/static/src/xml/snippets.xml

```diff
@@ -28,6 +28,7 @@
                         <div class="o_move_handle fa fa-arrows"/>
                     </div>
                     <div class="o_overlay_edit_options">
+                        <button type="button" class="o_snippet_replace o_we_bg_warning fa fa-refresh d-none"/>
```

Comment:
No need for `type="button"` on a `<button>`. (From an accessibility POV, there should be an `aria-label` here... but this button can't be reached in any manner anyway, so...)

Reply:
ok so let's leave it like that ;)

Reply:
Well, `type="button"` is still useless on a `<button>` :)